scripts/verify-cherrypick-pr.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+"""Script to verify a cherry-pick PR for a certain version
+
+The verification is done by comparing commits included in the PR against
+merged PRs targeting corresponding milestone and with "cherry-pick needed" label.
+"""
+from github import Github
+from github.Milestone import Milestone
+from github.Label import Label
+import sys
+import re
+
+
+def _get_milestone(repo, title):
+    for milestone in repo.get_milestones(state='all'):
+        if milestone.title == title:
+            return milestone
+
+    raise Exception('Milestone not found')
+
+def _get_label(repo):
+    for label in repo.get_labels():
+        if label.name == 'cherry-pick needed':
+            return label
+
+    raise Error('Label not found')
+
+
+def _main():
+    try:
+        token = sys.argv[1]
+    except IndexError:
+        sys.stderr.write('Please provide a valid GitHub access token\n')
+        sys.exit(1)
+
+    g = Github(token)
+    repo = g.get_repo('grafana/grafana')
+
+    milestone_name = '6.5.0'
+
+    milestone = _get_milestone(repo, milestone_name)
+    label = _get_label(repo)
+
+    pr_commits = {}
+    for issue in repo.get_issues(state='closed', milestone=milestone, labels=[label]):
```

Comment:
Additionally, the script could check if there are open issues in the milestone and report an error.

Reply:
Good point @papagian